src/main/kotlin/com/openlattice/organizations/ExternalDatabaseManagementService.kt

```diff
@@ -148,7 +229,7 @@ class ExternalDatabaseManagementService(
         val dbName = PostgresDatabases.buildOrganizationDatabaseName(orgId)
         val sql = "SELECT $columnNamesSql FROM $tableName LIMIT $rowCount"
         BasePostgresIterable(
-                StatementHolderSupplier(acm.connect(dbName), sql)
+                StatementHolderSupplier(externalDbManager.connect(dbName), sql)
```

Comment:
same comments as earlier for this and below about using the helper that just takes an org id instead of calling `PostgresDatabases.buildOrganizationDatabaseName(orgId)`